
geometry_msgs/Point position
geometry_msgs/Point min_position
geometry_msgs/Point max_position
geometry_msgs/Point normal
float64 radius
